app/common/resources/public/language/en.edn

```diff
@@ -952,7 +953,7 @@
   "Respondents connection to the land",
   :thk.contract/warranty-period "Warranty period",
   :land-acquisition/pos-number "POS#",
-  :company/country "Country",
+  :company/country "Country of registeration",
```

Comment:
```suggestion
  :company/country "Country of registration",
```

Reply:
Can't change the translations like this, but good catch.